src/D-FSL-1.0.xml

```diff
@@ -0,0 +1,380 @@
+<SPDX name="Deutsche Freie Software Lizenz" identifier="D-FSL-1.0" osi-approved="false">
+  <urls>
+    <url>http://www.dipp.nrw.de/d-fsl/lizenzen/</url>
+    <url>http://www.dipp.nrw.de/d-fsl/index_html/lizenzen/de/D-FSL-1_0_de.txt</url>
+    <url>http://www.dipp.nrw.de/d-fsl/index_html/lizenzen/en/D-FSL-1_0_en.txt</url>
+  </urls>
+  <notes>This license was created for and is backed by the German state. The English translation can be found here:
+         http://www.dipp.nrw.de/d-fsl/index_html/lizenzen/en/D-FSL-1_0_en.txt</notes>
+  <license>
+        <title>
+            <p>Deutsche Freie Software Lizenz</p>
+        </title>
+        <copyright>
+            <p>(c) Ministerium für Wissenschaft und Forschung Nordrhein-Westfalen 2004</p>
+        </copyright>
+        <optional>
+            <p>Erstellt von Axel Metzger und Till Jaeger, Institut für Rechtsfragen der Freien und Open Source Software
+             - (http://www.ifross.de).</p>
```

Comment:
Upstream is inconsistent here, using [parens in the HTML][1] and [`<>` in the text][2].  I recommend using `<alt>` to cover both cases, and have no opinion on which form should be canonical.

[1]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/deutsche-freie-software-lizenz
[2]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/D-FSL-1_0_de.txt/view